api/v1alpha1/network_types.go

```diff
@@ -44,9 +44,12 @@ type ProviderPropertiesStatus struct {
 
 // +kubebuilder:validation:MinLength:=1
 // +kubebuilder:validation:MaxLength:=255
+// +kubebuilder:validation:Pattern:="^[A-Za-z0-9]{1,63}(.[A-Za-z0-9-]{1,63})*(.[A-Za-z]{2,63})*.?$"
+
```

Comment:
The extra line broke our linter job, I wonder why this wasn't found in CI, and the PR prevented from merging.

Reply:
Ah, that's because the linter last ran before we merged https://github.com/k-orc/openstack-resource-controller/pull/195. If the PR had been rebased, we would have seen the failure.